docs/articles/documentation/recipes/pass-custom-command-line-parameters.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: docs
+title: Pass Custom Command Line Parameters
+permalink: /documentation/recipes/pass-custom-command-line-parameters.html
+---
+# Pass Custom Command Line Parameters
```

Comment:
I don't know the utility that uses the same approach. In my opinion, it is an anti-pattern. The customer's argument name may match with the build-in TestCafe's argument name for the next TestCafe major version. It can lead to unexpected issues.

Reply:
We've added it to FAQ for some reason:
https://devexpress.github.io/testcafe/faq/#how-do-i-work-with-configuration-files-and-environment-variables
We can reconsider this of course.